tests/sles4sap/hana_install.pm

```diff
@@ -15,23 +15,17 @@ use base 'sles4sap';
 use strict;
 use warnings;
 use testapi;
-use utils 'zypper_call';
+use utils qw(file_content_replace zypper_call);
 use version_utils 'is_sle';
 use POSIX 'ceil';
 
-sub upload_install_log {
-    script_run "tar -zcvf /tmp/hana_install.log.tgz /var/tmp/hdb*";
-    upload_logs "/tmp/hana_install.log.tgz";
-}
-
 sub run {
     my ($self) = @_;
     my ($proto, $path) = $self->fix_path(get_required_var('HANA'));
 
     my $sid      = get_required_var('INSTANCE_SID');
     my $instid   = get_required_var('INSTANCE_ID');
-    my $password = 'Qwerty_123';
-    set_var('PASSWORD', $password);
```

Comment:
Both `tests/sles4sap/hana_test` and `tests/sles4sap/hana_cluster` rely on the `PASSWORD` variable. Either keep the `set_var('PASSWORD', $password);` in this module, or also introduce the change from L28 in `tests/sles4sap/hana_cluster`